options.go

```diff
@@ -31,6 +35,23 @@ func defaultConfig() Config {
 		timeout:      30 * time.Second,
 
 		paramEncoders: map[reflect.Type]ParamEncoder{},
+
+		httpClient: &http.Client{
```

Comment:
I'd make this default point to the default global - we want a shared instance of `http.Transport` so that the limits apply between jsonrpc connections.

Reply:
Good call, updated!